index.js

```diff
@@ -24,6 +24,11 @@ exports.run = function(runner, specs) {
       debug('cucumber command: "' + cliArguments.join(' ') + '"');
 
       Cucumber.Cli(cliArguments).run(function (isSuccessful) {
+        if (!isSuccessful) {
+          reject(new Error('Cucumber has fail.'))
+          return
```

Comment:
`onComplete` must be called per the [adapter requirements](https://github.com/angular/protractor/blob/ddb8584a59343284904676ef6d8db5c1c996b900/lib/frameworks/README.md#requirements). This return short circuits that.